regen/warnings.pl

```diff
@@ -74,7 +74,15 @@ BEGIN
                                 'debugging'     => [ 5.008, DEFAULT_ON],
                                 'malloc'        => [ 5.008, DEFAULT_ON],
                            }],
-        'deprecated'    => [ 5.008, DEFAULT_ON],
+        'deprecated'    => [ 5.008, DEFAULT_ON, {
+                                'deprecated::goto_construct'           => [ 5.011003, DEFAULT_ON],
+                                'deprecated::unicode_property_name'    => [ 5.011003, DEFAULT_ON],
+                                'deprecated::dot_in_inc'               => [ 5.025011, DEFAULT_ON],
+                                'deprecated::version_downgrade'        => [ 5.035009, DEFAULT_ON],
+                                'deprecated::delimiter_will_be_paired' => [ 5.035010, DEFAULT_ON],
+                                'deprecated::apostrophe_as_pack_sep'   => [ 5.037009, DEFAULT_ON],
```

Comment:
"pack" might be confused with the `pack()` function. Perhaps better to use "pkg" or "package" spelled out in full?

Reply:
I wanted to avoid it getting too long, but given we kind /want/ to discourage people from disabling deprecation warnings (especially this one), I think in hindsight spelling it out makes a lot of sense.

Reply:
Done.